hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/AWSV4AuthValidator.java

```diff
@@ -39,6 +40,11 @@ final class AWSV4AuthValidator {
   private static final Logger LOG =
       LoggerFactory.getLogger(AWSV4AuthValidator.class);
   private static final String HMAC_SHA256_ALGORITHM = "HmacSHA256";
+  private static final Pattern STR_TO_SIGN_PATTERN =
+          Pattern.compile("^AWS4[A-Z\\-\\d]+\n" +
+                  "\\d{8}T\\d{6}Z\n" +
+                  "\\d{8}/[a-z\\-\\d]+/[a-zA-Z\\d]+/aws4_request\n" +
+                  "[a-fA-F\\d]+$");
```

Comment:
```suggestion
                  "[a-fA-F\\d]{64}$");
```

We could modify `[a-fA-F\\d]+` to `[a-fA-F\\d]{64}` to match exactly 64 characters of a hexadecimal string.